Re-export `errors` from `distutils` (#2858)
pypa/setuptools: docs/userguide/extension.rst

how setuptools' own commands are installed: the setuptools project's setup
script defines entry points for them!

.. note::
   When creating commands, and specially when defining custom ways of building

>    compiled extensions (for example via ``build_ext``), you might want to
>    handle exceptions such as ``CompileError``, ``LinkError``, ``LibError``,

Reviewer: I try to use imperative voice in docs.

```suggestion
   compiled extensions (for example via ``build_ext``), consider
   handling exceptions such as ``CompileError``, ``LinkError``, ``LibError``,
```
